src/circuit/gadget/ecc/chip.rs

```diff
@@ -142,7 +142,7 @@ pub struct EccConfig {
     add_incomplete: add_incomplete::Config,
 
     /// Complete addition
-    pub q_add: Selector,
+    add: add::Config,
```

Comment:
`add_config`

Reply:
I disagree with @daira's suggestion; it will be accessed as `config.add`, so `config.add_config` is redundant.

Reply:
In that case `_config` should be removed from the existing names that have it.

Reply:
I think the sub-sub-configs use `_config` in places, but the sub-configs don't. I agree we should be consistent.